packages/calypso-e2e/src/lib/components/editor-toolbar-component.ts

```diff
@@ -5,7 +5,7 @@ export type PreviewOptions = 'Desktop' | 'Mobile' | 'Tablet';
 const panel = 'div.interface-interface-skeleton__header';
 const selectors = {
 	// Block Inserter
-	blockInserterButton: `${ panel } button[aria-label="Toggle block inserter"]`,
+	blockInserterButton: `${ panel } button.edit-post-header-toolbar__inserter-toggle`,
```

Comment:
This is for i18n reasons - prior to the refactor the selector used was as shown above, but due to my changes in #61502 I ended up breaking i18n tests in `trunk`.